packages/test/local-server-stress-tests/src/stressDataObject.ts

```diff
@@ -279,13 +285,26 @@ export class DefaultStressDataObject extends StressDataObject {
 		this._locallyCreatedObjects.push(obj);
 	}
 
-	private stageControls: StageControls | undefined;
+	// eslint-disable-next-line import/no-deprecated
```

Comment:
Very minor, but this looks a lot better to me if you use `/* eslint-disable import/no-deprecated */` before/after these 3 lines rather than interleaving the `next-line` ones